apps/web/components/Charts/v2/renderers/TooltipRenderer.tsx

```diff
@@ -0,0 +1,59 @@
+import { Tooltip, TooltipProps } from 'recharts'
+
+import { theme } from '@island.is/island-ui/theme'
+
+import { formatValueForPresentation } from '../utils'
+
+export const CustomTooltipRenderer = (props: TooltipProps<string, number>) => {
+  const { active, payload } = props
+  const isActive = active && payload && payload.length
+
+  if (!isActive) {
+    return null
+  }
+
+  return (
+    <ul
+      style={{
+        backgroundColor: 'white',
```

Comment:
I'm wondering if this could be done via a vanilla extract className instead of being inlined 🤷